README.md

```diff
@@ -62,25 +62,26 @@ Artifacts are built and managed in the development environment via [Apache Maven
 mvn clean install
 ```
 
-# Running the SDO IoT Platform SDK demo
+# Running the SDO IoT Platform SDK Demo
 
-The SDO IoT Platform SDK demo can either be run manually, by running the binaries,
-or it can be run using Docker scripts. To run each component in separate
+The SDO IoT Platform SDK demo can either be runned manually, by running the binaries,
```

Comment:
`runned` is grammatically incorrect. It should be left to `run` or if we want to use past tense then probably it can be changed to `ran`. The original word `run` looks more appropriate.

Reply:
Updated. runnned => run